llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -16890,3 +16890,13 @@ SITargetLowering::lowerIdempotentRMWIntoFencedLoad(AtomicRMWInst *AI) const {
   AI->eraseFromParent();
   return LI;
 }
+
+bool SITargetLowering::hasAndNot(SDValue Op) const {
+  // Return false if the operation is divergent, as AND-NOT optimization
+  // requires uniform behavior across threads.
```

Comment:
More that it's a scalar only instruction

Reply:
I have already updated it.